packages/examples/packages/ethereum-provider/src/index.ts

```diff
@@ -38,12 +38,37 @@ async function getVersion() {
   return version;
 }
 
+/**
+ * Get the Ethereum accounts made available to the snap using the `ethereum` global. This essentially
```

Comment:
I see now I made a typo here 😅.

```suggestion
 * Get the Ethereum accounts made available to the snap using the `ethereum` global. This is essentially
```

Reply:
Can you update this in the other comment too?

Reply:
👍